src/app/log.py

```diff
@@ -26,13 +27,43 @@ def configure_logging():
                 "class": "logging.StreamHandler",
                 "formatter": "mozlog_json",
                 "stream": sys.stdout,
-            }
+            },
+            "null": {
+                "class": "logging.NullHandler",
+            },
         },
         "loggers": {
-            "request.summary": {"handlers": ["console"], "level": "INFO"},
-            "backoff": {"handlers": ["console"], "level": "INFO"},
-            "src.jbi": {"handlers": ["console"], "level": "DEBUG"},
+            "request.summary": {
+                "handlers": ["console"],
+                "level": logging.INFO,
+                "propagate": False,
+            },
+            "backoff": {
+                "handlers": ["console"],
+                "level": logging.INFO,
+                "propagate": False,
+            },
+            "src": {
+                "handlers": ["console"],
+                "level": settings.log_level.upper(),
```

Comment:
The `level` param (and on the following below) is not necessary, since they all use the `console` handler, which is already set to `settings.log_level.upper()`